frontend/src/metabase/admin/settings/components/widgets/SettingInput.tsx

```diff
@@ -10,7 +10,7 @@ const getValue = (value: string, type: string) => {
   return value;
 };
 
-interface SettingInputProps {
+export interface SettingInputProps {
```

Comment:
used in `frontend/src/metabase/admin/settings/components/widgets/NormalizedTextSettingInput/NormalizedTextSettingInput.tsx`